pages/config.md

```diff
@@ -1,16 +1,16 @@
 ---
 permalink: /updating-the-config-file/
-title: Updating the config file
+title: Update the config file
 ---
-Work your way through these steps to configure the template for your new guide
-by updating the `_config.yml` file:
+Work your way through these steps to update 
+the `_config.yml` file — this configures the 18F style template for your specific guide:
```

Comment:
@emileighoutlaw Does this new language make sense? Feel free to tweak it, of course.